packages/integration-tests/suites/replay/errors/errorMode/test.ts

```diff
@@ -98,7 +98,20 @@ sentryTest(
       expect.arrayContaining([
         {
           ...expectedClickBreadcrumb,
-          message: 'body > button#error',
```

Comment:
Removing this as it can leak attributes (e.g. aria label) that are masked with `maskAllText`